### PR TITLE
feat: add 3.0.3 and 3.0.4

### PR DESCRIPTION
**Pull request by diavrank**
**Changelog**

 - [3.0.4](https://docs.meteor.com/history.html#v3-0-4-2024-10-15)
 - [3.0.3](https://docs.meteor.com/history.html#v3-0-3-2024-09-11)

### CODE REVIEW

**Comment by mcorbelli** [comment]
Any news about closing the pull request?

**Comment by irafunesta** [comment]
Any news, this changes would be very useful for my project

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# IDEs folders
+.idea
```
Comment (GeoffreyBooth) [inline .gitignore:2]: Unrelated.

```suggestion
```

Comment (diavrank) [reply .gitignore]: For sure. This is done https://github.com/disney/meteor-base/pull/168/commits/1754aa13cf4f20596eb3b63e0a36fdd93bbf6d7b Thanks!

**Review by GeoffreyBooth — APPROVED** [review]
